order/buyer/receive/simple-order-event.md

```diff
@@ -0,0 +1,167 @@
+* `status.processStatus`: the order line's [Process status](#process-status).
+* `status.logisticsStatus`: the order line's [Logistics status](#logistics-status).
+* `eventDates`: some key line event date/times
+* `mergedItemDetails`: detailed part information provided by both buyer and supplier, see [Item details](#item-details).
+* `lastUpdatedAt`: is the latest date time the order line has been changed.
+
+### Buyer line
+
+`buyerLine` is an echo of your order line fields as explained in [Issue a new order](../issue/#lines)
+
+### Supplier line
+
+`supplierLine` contains the supplier order line fields:
+
+* `salesOrderNumber`: the sales order number as known in the supplier's ERP system
+* `salesOrderPosition`: the position within the supplier's sales order
+* `description`: a free format additional description of this line by the supplier
+* `requests`: the supplier can request different delivery schedule, prices and charge lines. Advised is to use the `statusLine.deliveryScheduleInclRequests` and `statusLine.pricesInclRequests` instead.
+* `properties`: are key-value based custom fields, added by the supplier
+* `notes`: are simple custom fields, added by the supplier
+* `documents`: contain meta data, objectId or url, of attached documents by the supplier.
+
+{% page-ref page="download-document.md" %}
+
+### Status line
+
+The status line represents the current order line values related to the current `Issued`, `Confirmed` or `InProgress` status. The `InclRequests` fields include the `InProgress` status and open buyer or supplier request values.
+
+#### Simple delivery schedule
+
+When using `simpleOrderEvent` the simple delivery schedule is used:
+
+* `scheduledDelivery`: the current delivery line, either having `Issued` or `Confirmed` values.
+* `scheduledDeliveryInclRequests`: the current delivery line, either having `Issued`, `InProgress` requests or `Confirmed` values.
+
+##### Scheduled delivery fields
+
+* `date`: the delivery date of this delivery line. Date has ISO 8601 date `yyyy-MM-dd` format. See also [Standards](../../api/standards.md).
+* `quantity`: the quantity of this delivery line. Quantity has a decimal `1234.56` format with any number of digits.
+
+##### Logistics fields
+
+These additional logistics fields are only available in the order line level delivery schedule:
+
+* `status`: the optional delivery line's [logistics status](#logistics-status).
+* `etd`: The optional logistics Estimated Time of Departure \(local date without time zone\). Date has ISO 8601 date `yyyy-MM-dd` format.
+* `eta`: The optional logistics Estimated Time of Arrival \(local date without time zone\). Date has ISO 8601 date `yyyy-MM-dd` format.
+
+#### Prices
+
+* `lines.prices`: the current prices, either having `Issued` or `Confirmed` values.
+* `lines.pricesInclRequests`: the current delivery line, either having `Issued`, `InProgress` requests or `Confirmed` values.
+
+##### Prices fields
```

Comment:
Same here; not clear that this is a sub-paragraph of Prices

Reply:
Improved